zipkin-collector-service/config/collector-cassandra.scala

```diff
@@ -25,11 +25,18 @@ import org.twitter.zipkin.storage.cassandra.ZipkinRetryPolicy
 val contactPoints: Array[String] = sys.env.get("CASSANDRA_CONTACT_POINTS").getOrElse("localhost")
   .split(",")
 
-val cluster = Cluster.builder()
-  .addContactPoints(contactPoints:_*)
-  .withSocketOptions(new SocketOptions().setConnectTimeoutMillis(10000).setReadTimeoutMillis(20000))
-  .withRetryPolicy(ZipkinRetryPolicy.INSTANCE)
-  .build()
+val cassandraUser: String = sys.env.get("CASSANDRA_USER").getOrElse("")
```

Comment:
I still think this is an awkward use of optional. instead of getOrElse(""), you could leave that off and check isPresent or whatever below.